debian/changelog

```diff
@@ -1,4 +1,4 @@
-pyndn (2.4b1-0.1) UNRELEASED; urgency=medium
+pyndn (2.4~beta1-1) UNRELEASED; urgency=medium
```

Comment:
That means our package version will not match the upstream version number (`2.4b1`). Do we want that?